ci-operator/step-registry/openshift/upgrade/aws/latestinstaller/openshift-upgrade-aws-latestinstaller-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-aws-latestinstaller
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-aws-pre
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: openshift-e2e-test-capabilities-check
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_TYPE: upgrade
+      TEST_SUITE: ""
+      TEST_UPGRADE_OPTIONS: ""
+      DELETE_MC: "false"
+  documentation: |-
+    The Openshift E2E AWS workflow executes the upgrade end-to-end test suite on AWS with a default cluster configuration and Loki as log collector, exercising an upgrade during the process.
```

Comment:
Perhaps you can update this comment that this workflow has to be used within installer repo?

Reply:
I added a note right below, not sure if its sufficient or need to be mentioned earlier